main.go

```diff
@@ -25,7 +25,8 @@ import (
 	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
 	"k8s.io/klog"
 	"k8s.io/klog/klogr"
-	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha2"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha3"
+	clusterv1alpha3 "sigs.k8s.io/cluster-api/api/v1alpha3"
```

Comment:
This looks like we are adding the v1alpha3 api to the scheme twice below, once for `clusterv1` and once for `clusterv1alpha3`

Reply:
https://github.com/kubernetes-sigs/cluster-api/pull/1530